examples/js/bitcom-example.js

```diff
@@ -0,0 +1,132 @@
+"use strict";
+
+// ----------------------------------------------------------------------------
+
+const ccxt = require("../../ccxt.js"),
+    log = require ('ololog').handleNodeErrors (),
+    asTable = require("as-table").configure({ delimiter: " | " });
+
+// ----------------------------------------------------------------------------
+
+function prettyJSONLog(str) {
+    log (JSON.stringify(str, null, 2))
+}
+
+(async () => {
+
+    const exchange = new ccxt.bitcom ({
+        verbose: process.argv.includes ('--verbose'),
+        timeout: 60000,
+        apiKey: 'ak-3d4dc159-8313-4726-860e-60355ccf1d24',
+        secret: 'A2I2HuUKqx7VnxQVvJhvowvmgQ3GgunDtdsHHyEjZoQXn14zmf8hB8MZzjlgGOFg'
```

Comment:
why are this keys hard-coded? I suggest they stay in an ENV variable.

Reply:
you can just replace your own keys here..